syntax = "proto3";

option java_package = "ru.test";

service Greeter {
  rpc SendNames (stream HelloRequest) returns (HelloReply);
  rpc GetResult (HelloRequest) returns (stream HelloReply);
}

message HelloRequest {
  string name = 1;
}

message HelloReply {
  string message = 1;
}
